component/src/main/java/org/cbioportal/genome_nexus/util/VariantId.java

```diff
@@ -0,0 +1,33 @@
+package org.cbioportal.genome_nexus.util;
+
+
+public class VariantId
+{
+    /**
+     * Build variant id in myVariantInfo id structure
+     *
+     * @param input input variant
+     * @return  variant id
+     */
+    public static String buildVariantId(String input)
```

Comment:
@leexgh The reason for the utility function is to make it general i.e. not specific to `MyVariantInfo`. There are two things that happen in this function:

1. Adding `chr`. Some genome builds indicate chromosomes without `chr` others without
2. Normalize `delACGinsT` to `delinsT` as per the Hgvs specification

then `buildRequest` could call the utility functions of HgvsUtil (or some name like that, since these are utilities related to Hgvs formatting):

```
String addChrPrefix(String hgvs)
String removeDeletedBases(String hgvs)
```

That way you can re-use those last two functions